src/box/lua/iproto.c

```diff
@@ -55,6 +75,20 @@ push_iproto_flag_constants(struct lua_State *L)
 	push_iproto_constant_subnamespace(L, "flag", flags, lengthof(flags));
 }
 
+/**
+ * Returns a string constant converted to lower case: the result is dynamically
+ * allocated using `xmalloc`.
+ */
+static char *
+constant_tolower(const char *constant, size_t len)
```

Comment:
Strange name. Maybe `string_tolower`? (or `string_tolower_copy` - more transparent, though longer)
And I would move this function to utils.

Reply:
Let's name it `to_lower_copy` like it is done in [boost](https://www.boost.org/doc/libs/1_70_0/doc/html/boost/algorithm/to_lower_copy.html).